app/code/Magento/InventoryInStorePickupQuote/etc/extension_attributes.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Api/etc/extension_attributes.xsd">
+    <extension_attributes for="Magento\Quote\Api\Data\AddressInterface">
+        <attribute code="pickup_location_code" type="string" />
+    </extension_attributes>
+</config>
```

Comment:
Please add the new line at the end of file

Reply:
It already in place)
Github do not show empty line :)